config/jobs/image-pushing/README.md

```diff
@@ -125,7 +125,7 @@ postsubmits:
   kubernetes-sigs/some-repo-name:
     # The name should be changed to match the repo name above
     - name: post-some-repo-name-push-images
-      cluster: test-infra-trusted
+      cluster: k8s-infra-prow-build-trusted
```

Comment:
there should really be a presubmit check enforcing which jobs can run on this cluster, same as the other trusted cluster
otherwise it's not trusted.

until then we shouldn't put anything important on it.